### ISSUE TITLE
IVPN  in servers.json doesn't include Australian wireguard servers

### ISSUE THREAD

**Issue by dcava** [qdm12/gluetun#1128]
Hi many thanks for this great container.

For some reason, the internal/storage/servers.json does not contain the wireguard addresses for Australian servers (all the other international wireguard servers seem to be there correctly and the OpenVPN servers are there). As far as I can tell the correct hostnames/addresses appear in the api endpoint that updater.go calls (https://api.ivpn.net/v4/servers/stats) and there doesn't appear to be anything strange about the address that would cause it to fail to parse.

For the time being, I'm just using the endpoints as a custom wireguard provider, which works fine, but would be great if the servers.json could reflect the current Australian wireguard servers.
Thanks

**Comment by qdm12** [comment]
Thanks for taking the time to create the issue 😉

I guess they added Australian wireguard servers recently, so the server information built-in is outdated on that.
You can update servers yourself with https://github.com/qdm12/gluetun/wiki/Updating-servers and that should add them to your persisted servers.json in `/gluetun/servers.json` which will take precedence over the ones built-in (due to the higher timestamp).

I'll close this for now, but please do let me know if it doesn't work and I'll re-open.

**Comment by dcava** [comment]
Apoliges, should have clarified that I run the update but it still doesn't seem to pick up the new servers?

```
/volume1/docker# docker run -it --rm -v /volume1/docker/gluetun:/gluetun qmcgaw/gluetun update -enduser -providers ivpn
2022-09-01T21:14:33Z INFO merging by most recent 11645 hardcoded servers and 11645 servers read from /gluetun/servers.json
2022-09-01T21:14:33Z INFO updating Ivpn servers...
```

Thanks

**Comment by qdm12** [comment]
Fixed in e59e28152f7ec63339b0fbd2a84145ef38ca070f 👍 
It totally [did miss a bunch of Wireguard servers](https://github.com/qdm12/gluetun/commit/e59e28152f7ec63339b0fbd2a84145ef38ca070f#diff-637eabbc0849b99e5719a9b46504d1e1d8d88d64cd3afb806b92dd0f95f65d62R29455), the only wireguard servers there were ones with no matching OpenVPN hostname, so very little compared to the actual total number of Wireguard servers!

Thanks again for reporting this!

**Comment by dcava** [comment]
Amazing turn around - thanks!
